dir.props

```diff
@@ -255,8 +255,10 @@
     <RuntimeProjectFile Condition="'$(RuntimeProjectFile)' == ''">$(ProjectDir)\external\runtime\runtime.depproj</RuntimeProjectFile>
     
     <!-- Paths to binplace package content -->
-    <NETCoreAppPackageRefPath>$(BinDir)netcoreapp\pkg\ref</NETCoreAppPackageRefPath>
-    <NETCoreAppPackageRuntimePath>$(BinDir)netcoreapp\pkg\lib</NETCoreAppPackageRuntimePath>
+    <NETCoreAppPackageRefPath>$(BinDir)pkg\netcoreapp\ref</NETCoreAppPackageRefPath>
```

Comment:
I've been trying to address this, but I'm having problems with the binplacing logic. The reason is that today we use the Target group specific paths in the following snippet:
```xml
    <BinplaceConfiguration Condition="'$(IsNETCoreApp)' == 'true'" Include="netcoreapp-$(OSGroup)">
      <RefPath Condition="'$(IsNETCoreAppRef)' == 'true'">$(NETCoreAppPackageRefPath)</RefPath>
      <RuntimePath>$(NETCoreAppPackageRuntimePath)</RuntimePath>
    </BinplaceConfiguration>
    <BinplaceConfiguration Condition="'$(IsUAP)' == 'true'" Include="uap-$(OSGroup)">
      <RefPath Condition="'$(IsUAPRef)'=='true'">$(UAPPackageRefPath)</RefPath>
      <RuntimePath>$(UAPPackageRuntimePath)</RuntimePath>
    </BinplaceConfiguration>
```

Basically the problem is that if we are building a netstandard asset, today we are binplacing it in both the uap package paths and the netcoreapp ones. If we decide to unify these paths, then we would be giving that up unless we change the logic of how we do that here.